zulipterminal/ui_tools/views.py

```diff
@@ -838,13 +867,7 @@ def __init__(self, controller: Any,
 
 
 class HelpView(PopUpView):
-    def __init__(self, controller: Any) -> None:
-        widths = [(len(binding['help_text']) + 4,
-                   len(", ".join(binding['keys'])))
-                  for binding in KEY_BINDINGS.values()]
-        max_widths = [max(width) for width in zip(*widths)]
-        self.width = sum(max_widths)
-
+    def __init__(self, controller: Any, popup_title: str) -> None:
```

Comment:
Do we actually need to pass in the title? Didn't you settle on the length, previously? This is like a `minimum_width` parameter. That said, I see the appeal of just expecting the title, as then we can manage the extra spacing internally (eg. the +4)

Reply:
The reason I didn't go with `minimum_width` is that even when we pass `minimum_width=len(popup_title)`, we need the +4 in the `table_widths` method which doesn't justify the 'minimum width' name?

An alternative could be passing `len(popup_title) + 4` but that wouldn't be clear either (why we're adding +4 to the length?). To address that, we could add a comment but that'd require adding a comment before every call.

The approach where we handle the +4 internally allows adding a single comment.

I could use a suggestion about how to approach this.